### ISSUE TITLE
Support of sybil APIs and P2P-level security tests for native bitcoin client

### ISSUE THREAD

**Issue by ygnkim** [kaistshadow/blockchain-sim#128]
This feature is targeted for [release 0.1.0](https://github.com/kaistshadow/blockchain-sim/milestone/2).

P2P network is crucial for the security of bitcoin system. 
We want to provide basic facilities for testing bitcoin system for recent P2P-level security attacks (Eclipse attack, and Erebus attack). 
The basic facilities include :
- Emulation support for recent bitcoin client (v0.19.1)
- General APIs for launching a large-scale sybil node network
- General APIs for launching a large-scale bitcoin sybil network
- Use-case (instantiation) of bitcoin sybil network APIs (i.e., Eclipse attack, and Erebus attack)

Currently, APIs layer for sybil network are implemented by the modification of Shadow and the monolithic attacker node(shadow-ISP/3_bitcoin/emulated-benigns-realistic.cpp).
Specifically, APIs for general sybil network and APIs for 'bitcoin' sybil network are implemented cohesively.

Before refactoring the current APIs and clearly separating the layers of APIs, I want to create more samples.
In other words, I want to test sybil network using a simple testing node (not bitcoin node).

Thus, the very first task is to design & implement a simple network application.
